internal/status/ing_status.go

```diff
@@ -217,14 +217,14 @@ func updateIngAnnotations(ingObj *networkingv1.Ingress, hostnamesToBeUpdated []s
 			delete(vsAnnotations, k)
 		}
 	}
-
+	tenant := lib.GetTenantInNamespace(ingObj.Namespace)
 	// compare the vs annotations for this ingress object
```

Comment:
I think it will be better to add Tenant info in UpdateOptions instead of fetching here - we are already adding VS specific info to UpdateOptions.

Reply:
We are using UpdateOptions everywhere, having tenant info in UpdateOptions will eliminate the need of fetching Tenant from namespace everywhere

Reply:
I will add the tenant field in updateOptions. Was planning same thing earlier.